CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ appear at the top.
 ## [Unreleased][]
 
   * Your contribution here!
+  * [#397](https://github.com/capistrano/sshkt/pull/397): Fix NoMethodError assign_defaults with net-ssh older than 4.0.0 - [@shirosaki](https://github.com/shirosaki)
```

Comment:
Ah, I see there is a typo `sshkt` in the URL. I'll fix it in master. Good 👀 @reedloden